spec/pagseguro/extensions/collection_object_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+describe PagSeguro::Extensions::CollectionObject do
+  before(:all) do
+    # Collection
+    class MyItems
+      include PagSeguro::Extensions::CollectionObject
+    end
+
+    # Item
+    class MyItem
+      attr_accessor :name
+
+      def ==(other)
+        name.eql?(other.name)
+      end
+    end
+  end
+
+  let(:collection) do
+    MyItems.new
+  end
+
+  let(:object) do
+    item = MyItem.new
+    item.name = 'Item1'
+    item
+  end
+
+  context "include module" do
+    it "Enumerable" do
+      expect(described_class.included_modules).to include(Enumerable)
```

Comment:
better than verify included modules, is verify if the object behaves the way you want, duck type